content/spin/cache.md

```diff
@@ -0,0 +1,64 @@
+title = "Spin Cache"
+template = "spin_main"
+date = "2022-03-14T00:22:56Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/spin/cache.md"
+
+---
+
+- [The Spin Registry Cache](#the-spin-registry-cache)
+- [Viewing Registry Cache Files](#viewing-registry-cache-files)
+- [Clearing Registry Cache Files](#clearing-registry-cache-files)
+
+## The Spin Registry Cache
+
+A running Spin application can [fetch resources from remote registries](https://developer.fermyon.com/spin/spin-oci) and individual component sources via HTTP endpoints. These resources naturally consume network bandwidth. To ensure network efficiency, and to prevent waiting to download the same files every time an application is started, Spin automatically maintains a local `spin/registry` cache.
```

Comment:
Link goes to the tutorial rather than the detail material; is that intentional?  (The backlink is from the detail material.)

Reply:
Yes, that was intentional.